docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -502,6 +507,28 @@ For FQDN reporting to work as expected, the hostname of the current host must ei
 
 If neither pre-requisite is satisfied, `host.name` continues to report the hostname of the current host in a non-fully-qualified format.
 
+
+[discrete]
+[[fleet-agent-unenrollment-timeout]]
+== Set an unenrollment timeout for inactive agents
+
+You can configure a length of time after which any inactive {agent}s are automatically unenrolled and their API keys invalidated.
+This setting is useful when you have agents running in an ephemeral environment, such as Docker or Kubernetes, and you want to prevent inactive agents from consuming unused API keys.
```

Comment:
```suggestion
This setting is useful when you have agents running in an ephemeral environment, such as Docker or {k8s}, and you want to prevent inactive agents from consuming unused API keys.
```